2021-08-03 15:26:05,692:INFO:PyCaret Supervised Module
2021-08-03 15:26:05,692:INFO:ML Usecase: classification
2021-08-03 15:26:05,692:INFO:version 2.3.3
2021-08-03 15:26:05,692:INFO:Initializing setup()
2021-08-03 15:26:05,693:INFO:setup(target=Risk, ml_usecase=classification, available_plots={'parameter': 'Hyperparameters', 'auc': 'AUC', 'confusion_matrix': 'Confusion Matrix', 'threshold': 'Threshold', 'pr': 'Precision Recall', 'error': 'Prediction Error', 'class_report': 'Class Report', 'rfe': 'Feature Selection', 'learning': 'Learning Curve', 'manifold': 'Manifold Learning', 'calibration': 'Calibration Curve', 'vc': 'Validation Curve', 'dimension': 'Dimensions', 'feature': 'Feature Importance', 'feature_all': 'Feature Importance (All)', 'boundary': 'Decision Boundary', 'lift': 'Lift Chart', 'gain': 'Gain Chart', 'tree': 'Decision Tree', 'ks': 'KS Statistic Plot'}, train_size=0.7, test_data=None, preprocess=True, imputation_type=simple, iterative_imputation_iters=5, categorical_features=None, categorical_imputation=constant, categorical_iterative_imputer=lightgbm, ordinal_features=None, high_cardinality_features=None, high_cardinality_method=frequency, numeric_features=None, numeric_imputation=mean, numeric_iterative_imputer=lightgbm, date_features=None, ignore_features=None, normalize=False, normalize_method=zscore, transformation=False, transformation_method=yeo-johnson, handle_unknown_categorical=True, unknown_categorical_method=least_frequent, pca=False, pca_method=linear, pca_components=None, ignore_low_variance=False, combine_rare_levels=False, rare_level_threshold=0.1, bin_numeric_features=None, remove_outliers=False, outliers_threshold=0.05, remove_multicollinearity=False, multicollinearity_threshold=0.9, remove_perfect_collinearity=True, create_clusters=False, cluster_iter=20, polynomial_features=False, polynomial_degree=2, trigonometry_features=False, polynomial_threshold=0.1, group_features=None, group_names=None, feature_selection=False, feature_selection_threshold=0.8, feature_selection_method=classic, feature_interaction=False, feature_ratio=False, interaction_threshold=0.01, fix_imbalance=False, fix_imbalance_method=None, transform_target=False, transform_target_method=box-cox, data_split_shuffle=True, data_split_stratify=False, fold_strategy=stratifiedkfold, fold=10, fold_shuffle=False, fold_groups=None, n_jobs=-1, use_gpu=False, custom_pipeline=None, html=True, session_id=None, log_experiment=False, experiment_name=None, log_plots=False, log_profile=False, log_data=False, silent=False, verbose=True, profile=False, profile_kwargs=None, display=None)
2021-08-03 15:26:05,693:INFO:Checking environment
2021-08-03 15:26:05,693:INFO:python_version: 3.8.8
2021-08-03 15:26:05,693:INFO:python_build: ('default', 'Apr 13 2021 12:59:45')
2021-08-03 15:26:05,693:INFO:machine: x86_64
2021-08-03 15:26:05,702:INFO:platform: macOS-10.16-x86_64-i386-64bit
2021-08-03 15:26:05,703:INFO:Memory: svmem(total=17179869184, available=10104987648, percent=41.2, used=5589364736, free=6918664192, active=3153096704, inactive=2596245504, wired=2436268032)
2021-08-03 15:26:05,703:INFO:Physical Core: 8
2021-08-03 15:26:05,703:INFO:Logical Core: 16
2021-08-03 15:26:05,703:INFO:Checking libraries
2021-08-03 15:26:05,703:INFO:pd==1.2.4
2021-08-03 15:26:05,703:INFO:numpy==1.19.5
2021-08-03 15:26:05,703:INFO:sklearn==0.23.2
2021-08-03 15:26:05,703:INFO:lightgbm==3.2.1
2021-08-03 15:26:05,734:INFO:catboost==0.26
2021-08-03 15:26:05,763:INFO:xgboost==1.3.3
2021-08-03 15:26:06,003:INFO:mlflow==1.19.0
2021-08-03 15:26:06,003:INFO:Checking Exceptions
2021-08-03 15:26:06,004:INFO:Declaring global variables
2021-08-03 15:26:06,004:INFO:USI: 75f3
2021-08-03 15:26:06,004:INFO:pycaret_globals: {'gpu_param', 'transform_target_param', 'master_model_container', 'stratify_param', 'fold_generator', 'seed', 'exp_name_log', 'fix_imbalance_method_param', 'fold_param', 'display_container', 'USI', 'imputation_classifier', 'n_jobs_param', '_available_plots', 'data_before_preprocess', 'fold_shuffle_param', 'X_test', 'X', 'y_train', 'prep_pipe', 'html_param', 'y_test', 'imputation_regressor', 'logging_param', 'fold_groups_param', '_ml_usecase', '_internal_pipeline', 'transform_target_method_param', 'fix_imbalance_param', 'pycaret_globals', 'target_param', '_all_models_internal', 'experiment__', 'y', '_all_models', 'fold_groups_param_full', '_all_metrics', '_gpu_n_jobs_param', 'log_plots_param', 'X_train', 'iterative_imputation_iters_param', 'create_model_container'}
2021-08-03 15:26:06,004:INFO:Preparing display monitor
2021-08-03 15:26:06,004:INFO:Preparing display monitor
2021-08-03 15:26:06,016:INFO:Importing libraries
2021-08-03 15:26:06,016:INFO:Copying data for preprocessing
2021-08-03 15:26:06,023:INFO:Declaring preprocessing parameters
2021-08-03 15:26:06,024:INFO:Creating preprocessing pipeline
2021-08-03 15:26:06,027:INFO:Preprocessing pipeline created successfully
2021-08-03 15:26:06,027:ERROR:(Process Exit): setup has been interupted with user command 'quit'. setup must rerun.
2021-08-03 15:26:06,028:INFO:Creating global containers
2021-08-03 15:26:06,029:INFO:Internal pipeline: Pipeline(memory=None, steps=[('empty_step', 'passthrough')], verbose=False)
2021-08-03 15:26:07,918:INFO:Creating grid variables
2021-08-03 15:26:07,935:INFO:create_model_container: 0
2021-08-03 15:26:07,935:INFO:master_model_container: 0
2021-08-03 15:26:07,935:INFO:display_container: 1
2021-08-03 15:26:07,940:INFO:Pipeline(memory=None,
         steps=[('dtypes',
                 DataTypes_Auto_infer(categorical_features=[],
                                      display_types=True, features_todrop=[],
                                      id_columns=[],
                                      ml_usecase='classification',
                                      numerical_features=[], target='Risk',
                                      time_features=[])),
                ('imputer',
                 Simple_Imputer(categorical_strategy='not_available',
                                fill_value_categorical=None,
                                fill_value_numerical=None,
                                numeric_strateg...
                ('scaling', 'passthrough'), ('P_transform', 'passthrough'),
                ('binn', 'passthrough'), ('rem_outliers', 'passthrough'),
                ('cluster_all', 'passthrough'),
                ('dummy', Dummify(target='Risk')),
                ('fix_perfect', Remove_100(target='Risk')),
                ('clean_names', Clean_Colum_Names()),
                ('feature_select', 'passthrough'), ('fix_multi', 'passthrough'),
                ('dfs', 'passthrough'), ('pca', 'passthrough')],
         verbose=False)
2021-08-03 15:26:07,940:INFO:setup() succesfully completed......................................
2021-08-03 15:26:41,156:INFO:Initializing create_model()
2021-08-03 15:26:41,157:INFO:create_model(estimator=dt, fold=None, round=4, cross_validation=True, predict=True, fit_kwargs=None, groups=None, refit=True, verbose=True, system=True, metrics=None, display=None, kwargs={})
2021-08-03 15:26:41,157:INFO:Checking exceptions
2021-08-03 15:26:41,157:INFO:Preparing display monitor
2021-08-03 15:26:41,176:INFO:Importing libraries
2021-08-03 15:26:41,176:INFO:Copying training dataset
2021-08-03 15:26:41,176:INFO:Defining folds
2021-08-03 15:26:41,177:INFO:Declaring metric variables
2021-08-03 15:26:41,184:INFO:Importing untrained model
2021-08-03 15:26:41,193:INFO:Decision Tree Classifier Imported succesfully
2021-08-03 15:26:41,208:INFO:Starting cross validation
2021-08-03 15:26:41,209:INFO:Cross validating with StratifiedKFold(n_splits=10, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:27:58,574:INFO:PyCaret Supervised Module
2021-08-03 15:27:58,574:INFO:ML Usecase: classification
2021-08-03 15:27:58,574:INFO:version 2.3.3
2021-08-03 15:27:58,574:INFO:Initializing setup()
2021-08-03 15:27:58,574:INFO:setup(target=Risk, ml_usecase=classification, available_plots={'parameter': 'Hyperparameters', 'auc': 'AUC', 'confusion_matrix': 'Confusion Matrix', 'threshold': 'Threshold', 'pr': 'Precision Recall', 'error': 'Prediction Error', 'class_report': 'Class Report', 'rfe': 'Feature Selection', 'learning': 'Learning Curve', 'manifold': 'Manifold Learning', 'calibration': 'Calibration Curve', 'vc': 'Validation Curve', 'dimension': 'Dimensions', 'feature': 'Feature Importance', 'feature_all': 'Feature Importance (All)', 'boundary': 'Decision Boundary', 'lift': 'Lift Chart', 'gain': 'Gain Chart', 'tree': 'Decision Tree', 'ks': 'KS Statistic Plot'}, train_size=0.7, test_data=None, preprocess=True, imputation_type=simple, iterative_imputation_iters=5, categorical_features=None, categorical_imputation=constant, categorical_iterative_imputer=lightgbm, ordinal_features=None, high_cardinality_features=None, high_cardinality_method=frequency, numeric_features=None, numeric_imputation=mean, numeric_iterative_imputer=lightgbm, date_features=None, ignore_features=None, normalize=False, normalize_method=zscore, transformation=False, transformation_method=yeo-johnson, handle_unknown_categorical=True, unknown_categorical_method=least_frequent, pca=False, pca_method=linear, pca_components=None, ignore_low_variance=False, combine_rare_levels=False, rare_level_threshold=0.1, bin_numeric_features=None, remove_outliers=False, outliers_threshold=0.05, remove_multicollinearity=False, multicollinearity_threshold=0.9, remove_perfect_collinearity=True, create_clusters=False, cluster_iter=20, polynomial_features=False, polynomial_degree=2, trigonometry_features=False, polynomial_threshold=0.1, group_features=None, group_names=None, feature_selection=False, feature_selection_threshold=0.8, feature_selection_method=classic, feature_interaction=False, feature_ratio=False, interaction_threshold=0.01, fix_imbalance=False, fix_imbalance_method=None, transform_target=False, transform_target_method=box-cox, data_split_shuffle=True, data_split_stratify=False, fold_strategy=stratifiedkfold, fold=3, fold_shuffle=False, fold_groups=None, n_jobs=-1, use_gpu=False, custom_pipeline=None, html=True, session_id=None, log_experiment=False, experiment_name=None, log_plots=False, log_profile=False, log_data=False, silent=False, verbose=True, profile=False, profile_kwargs=None, display=None)
2021-08-03 15:27:58,574:INFO:Checking environment
2021-08-03 15:27:58,574:INFO:python_version: 3.8.8
2021-08-03 15:27:58,574:INFO:python_build: ('default', 'Apr 13 2021 12:59:45')
2021-08-03 15:27:58,574:INFO:machine: x86_64
2021-08-03 15:27:58,574:INFO:platform: macOS-10.16-x86_64-i386-64bit
2021-08-03 15:27:58,574:INFO:Memory: svmem(total=17179869184, available=10085969920, percent=41.3, used=5608423424, free=6848020480, active=3174322176, inactive=2609225728, wired=2434101248)
2021-08-03 15:27:58,574:INFO:Physical Core: 8
2021-08-03 15:27:58,575:INFO:Logical Core: 16
2021-08-03 15:27:58,575:INFO:Checking libraries
2021-08-03 15:27:58,575:INFO:pd==1.2.4
2021-08-03 15:27:58,575:INFO:numpy==1.19.5
2021-08-03 15:27:58,575:INFO:sklearn==0.23.2
2021-08-03 15:27:58,575:INFO:lightgbm==3.2.1
2021-08-03 15:27:58,575:INFO:catboost==0.26
2021-08-03 15:27:58,575:INFO:xgboost==1.3.3
2021-08-03 15:27:58,575:INFO:mlflow==1.19.0
2021-08-03 15:27:58,575:INFO:Checking Exceptions
2021-08-03 15:27:58,576:INFO:Declaring global variables
2021-08-03 15:27:58,576:INFO:USI: 12ef
2021-08-03 15:27:58,576:INFO:pycaret_globals: {'gpu_param', 'transform_target_param', 'master_model_container', 'stratify_param', 'fold_generator', 'seed', 'exp_name_log', 'fix_imbalance_method_param', 'fold_param', 'display_container', 'USI', 'imputation_classifier', 'n_jobs_param', '_available_plots', 'data_before_preprocess', 'fold_shuffle_param', 'X_test', 'X', 'y_train', 'prep_pipe', 'html_param', 'y_test', 'imputation_regressor', 'logging_param', 'fold_groups_param', '_ml_usecase', '_internal_pipeline', 'transform_target_method_param', 'fix_imbalance_param', 'pycaret_globals', 'target_param', '_all_models_internal', 'experiment__', 'y', '_all_models', 'fold_groups_param_full', '_all_metrics', '_gpu_n_jobs_param', 'log_plots_param', 'X_train', 'iterative_imputation_iters_param', 'create_model_container'}
2021-08-03 15:27:58,576:INFO:Preparing display monitor
2021-08-03 15:27:58,576:INFO:Preparing display monitor
2021-08-03 15:27:58,590:INFO:Importing libraries
2021-08-03 15:27:58,590:INFO:Copying data for preprocessing
2021-08-03 15:27:58,598:INFO:Declaring preprocessing parameters
2021-08-03 15:27:58,599:INFO:Creating preprocessing pipeline
2021-08-03 15:27:58,604:INFO:Preprocessing pipeline created successfully
2021-08-03 15:27:58,604:ERROR:(Process Exit): setup has been interupted with user command 'quit'. setup must rerun.
2021-08-03 15:27:58,604:INFO:Creating global containers
2021-08-03 15:27:58,605:INFO:Internal pipeline: Pipeline(memory=None, steps=[('empty_step', 'passthrough')], verbose=False)
2021-08-03 15:28:01,686:INFO:Creating grid variables
2021-08-03 15:28:01,699:INFO:create_model_container: 0
2021-08-03 15:28:01,699:INFO:master_model_container: 0
2021-08-03 15:28:01,699:INFO:display_container: 1
2021-08-03 15:28:01,703:INFO:Pipeline(memory=None,
         steps=[('dtypes',
                 DataTypes_Auto_infer(categorical_features=[],
                                      display_types=True, features_todrop=[],
                                      id_columns=[],
                                      ml_usecase='classification',
                                      numerical_features=[], target='Risk',
                                      time_features=[])),
                ('imputer',
                 Simple_Imputer(categorical_strategy='not_available',
                                fill_value_categorical=None,
                                fill_value_numerical=None,
                                numeric_strateg...
                ('scaling', 'passthrough'), ('P_transform', 'passthrough'),
                ('binn', 'passthrough'), ('rem_outliers', 'passthrough'),
                ('cluster_all', 'passthrough'),
                ('dummy', Dummify(target='Risk')),
                ('fix_perfect', Remove_100(target='Risk')),
                ('clean_names', Clean_Colum_Names()),
                ('feature_select', 'passthrough'), ('fix_multi', 'passthrough'),
                ('dfs', 'passthrough'), ('pca', 'passthrough')],
         verbose=False)
2021-08-03 15:28:01,703:INFO:setup() succesfully completed......................................
2021-08-03 15:28:05,149:INFO:Initializing create_model()
2021-08-03 15:28:05,149:INFO:create_model(estimator=dt, fold=None, round=4, cross_validation=True, predict=True, fit_kwargs=None, groups=None, refit=True, verbose=True, system=True, metrics=None, display=None, kwargs={})
2021-08-03 15:28:05,149:INFO:Checking exceptions
2021-08-03 15:28:05,149:INFO:Preparing display monitor
2021-08-03 15:28:05,166:INFO:Importing libraries
2021-08-03 15:28:05,166:INFO:Copying training dataset
2021-08-03 15:28:05,167:INFO:Defining folds
2021-08-03 15:28:05,167:INFO:Declaring metric variables
2021-08-03 15:28:05,175:INFO:Importing untrained model
2021-08-03 15:28:05,183:INFO:Decision Tree Classifier Imported succesfully
2021-08-03 15:28:05,198:INFO:Starting cross validation
2021-08-03 15:28:05,199:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:28:07,052:INFO:Calculating mean and std
2021-08-03 15:28:07,055:INFO:Creating metrics dataframe
2021-08-03 15:28:07,073:INFO:Finalizing model
2021-08-03 15:28:07,079:INFO:Uploading results into container
2021-08-03 15:28:07,079:INFO:Uploading model into container now
2021-08-03 15:28:07,096:INFO:create_model_container: 1
2021-08-03 15:28:07,096:INFO:master_model_container: 1
2021-08-03 15:28:07,097:INFO:display_container: 2
2021-08-03 15:28:07,097:INFO:DecisionTreeClassifier(ccp_alpha=0.0, class_weight=None, criterion='gini',
                       max_depth=None, max_features=None, max_leaf_nodes=None,
                       min_impurity_decrease=0.0, min_impurity_split=None,
                       min_samples_leaf=1, min_samples_split=2,
                       min_weight_fraction_leaf=0.0, presort='deprecated',
                       random_state=3723, splitter='best')
2021-08-03 15:28:07,097:INFO:create_model() succesfully completed......................................
2021-08-03 15:28:29,312:INFO:Initializing tune_model()
2021-08-03 15:28:29,312:INFO:tune_model(estimator=DecisionTreeClassifier(ccp_alpha=0.0, class_weight=None, criterion='gini',
                       max_depth=None, max_features=None, max_leaf_nodes=None,
                       min_impurity_decrease=0.0, min_impurity_split=None,
                       min_samples_leaf=1, min_samples_split=2,
                       min_weight_fraction_leaf=0.0, presort='deprecated',
                       random_state=3723, splitter='best'), fold=None, round=4, n_iter=10, custom_grid=None, optimize=Accuracy, custom_scorer=None, search_library=scikit-learn, search_algorithm=None, early_stopping=False, early_stopping_max_iters=10, choose_better=False, fit_kwargs=None, groups=None, return_tuner=False, verbose=True, tuner_verbose=True, display=None, kwargs={})
2021-08-03 15:28:29,312:INFO:Checking exceptions
2021-08-03 15:28:29,314:INFO:Preparing display monitor
2021-08-03 15:28:29,332:INFO:Copying training dataset
2021-08-03 15:28:29,333:INFO:Checking base model
2021-08-03 15:28:29,333:INFO:Base model : Decision Tree Classifier
2021-08-03 15:28:29,342:INFO:Declaring metric variables
2021-08-03 15:28:29,350:INFO:Defining Hyperparameters
2021-08-03 15:28:29,420:INFO:Tuning with n_jobs=-1
2021-08-03 15:28:29,421:INFO:Initializing RandomizedSearchCV
2021-08-03 15:28:31,490:INFO:best_params: {'actual_estimator__min_samples_split': 9, 'actual_estimator__min_samples_leaf': 4, 'actual_estimator__min_impurity_decrease': 0.5, 'actual_estimator__max_features': 1.0, 'actual_estimator__max_depth': 2, 'actual_estimator__criterion': 'entropy'}
2021-08-03 15:28:31,492:INFO:Hyperparameter search completed
2021-08-03 15:28:31,492:INFO:SubProcess create_model() called ==================================
2021-08-03 15:28:31,493:INFO:Initializing create_model()
2021-08-03 15:28:31,494:INFO:create_model(estimator=DecisionTreeClassifier(ccp_alpha=0.0, class_weight=None, criterion='gini',
                       max_depth=None, max_features=None, max_leaf_nodes=None,
                       min_impurity_decrease=0.0, min_impurity_split=None,
                       min_samples_leaf=1, min_samples_split=2,
                       min_weight_fraction_leaf=0.0, presort='deprecated',
                       random_state=3723, splitter='best'), fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=True, verbose=True, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb47a2a7ee0>, kwargs={'min_samples_split': 9, 'min_samples_leaf': 4, 'min_impurity_decrease': 0.5, 'max_features': 1.0, 'max_depth': 2, 'criterion': 'entropy'})
2021-08-03 15:28:31,494:INFO:Checking exceptions
2021-08-03 15:28:31,494:INFO:Importing libraries
2021-08-03 15:28:31,494:INFO:Copying training dataset
2021-08-03 15:28:31,496:INFO:Defining folds
2021-08-03 15:28:31,496:INFO:Declaring metric variables
2021-08-03 15:28:31,520:INFO:Importing untrained model
2021-08-03 15:28:31,520:INFO:Declaring custom model
2021-08-03 15:28:31,545:INFO:Decision Tree Classifier Imported succesfully
2021-08-03 15:28:31,592:INFO:Starting cross validation
2021-08-03 15:28:31,593:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:28:32,708:INFO:Calculating mean and std
2021-08-03 15:28:32,708:INFO:Creating metrics dataframe
2021-08-03 15:28:32,717:INFO:Finalizing model
2021-08-03 15:28:32,721:INFO:Uploading results into container
2021-08-03 15:28:32,722:INFO:Uploading model into container now
2021-08-03 15:28:32,722:INFO:create_model_container: 2
2021-08-03 15:28:32,722:INFO:master_model_container: 2
2021-08-03 15:28:32,722:INFO:display_container: 3
2021-08-03 15:28:32,722:INFO:DecisionTreeClassifier(ccp_alpha=0.0, class_weight=None, criterion='entropy',
                       max_depth=2, max_features=1.0, max_leaf_nodes=None,
                       min_impurity_decrease=0.5, min_impurity_split=None,
                       min_samples_leaf=4, min_samples_split=9,
                       min_weight_fraction_leaf=0.0, presort='deprecated',
                       random_state=3723, splitter='best')
2021-08-03 15:28:32,722:INFO:create_model() succesfully completed......................................
2021-08-03 15:28:32,789:INFO:SubProcess create_model() end ==================================
2021-08-03 15:28:32,805:INFO:create_model_container: 2
2021-08-03 15:28:32,805:INFO:master_model_container: 2
2021-08-03 15:28:32,805:INFO:display_container: 3
2021-08-03 15:28:32,806:INFO:DecisionTreeClassifier(ccp_alpha=0.0, class_weight=None, criterion='entropy',
                       max_depth=2, max_features=1.0, max_leaf_nodes=None,
                       min_impurity_decrease=0.5, min_impurity_split=None,
                       min_samples_leaf=4, min_samples_split=9,
                       min_weight_fraction_leaf=0.0, presort='deprecated',
                       random_state=3723, splitter='best')
2021-08-03 15:28:32,806:INFO:tune_model() succesfully completed......................................
2021-08-03 15:28:51,584:INFO:Initializing compare_models()
2021-08-03 15:28:51,584:INFO:compare_models(include=None, fold=None, round=4, cross_validation=True, sort=Accuracy, n_select=3, budget_time=None, turbo=True, errors=ignore, fit_kwargs=None, groups=None, verbose=True, display=None, exclude=None)
2021-08-03 15:28:51,585:INFO:Checking exceptions
2021-08-03 15:28:51,585:INFO:Preparing display monitor
2021-08-03 15:28:51,585:INFO:Preparing display monitor
2021-08-03 15:28:51,612:INFO:Initializing Logistic Regression
2021-08-03 15:28:51,612:INFO:Total runtime is 3.0159950256347657e-06 minutes
2021-08-03 15:28:51,620:INFO:SubProcess create_model() called ==================================
2021-08-03 15:28:51,621:INFO:Initializing create_model()
2021-08-03 15:28:51,621:INFO:create_model(estimator=lr, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:28:51,621:INFO:Checking exceptions
2021-08-03 15:28:51,621:INFO:Importing libraries
2021-08-03 15:28:51,621:INFO:Copying training dataset
2021-08-03 15:28:51,622:INFO:Defining folds
2021-08-03 15:28:51,622:INFO:Declaring metric variables
2021-08-03 15:28:51,629:INFO:Importing untrained model
2021-08-03 15:28:51,635:INFO:Logistic Regression Imported succesfully
2021-08-03 15:28:51,649:INFO:Starting cross validation
2021-08-03 15:28:51,650:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:28:51,694:INFO:Calculating mean and std
2021-08-03 15:28:51,695:INFO:Creating metrics dataframe
2021-08-03 15:28:51,699:INFO:Uploading results into container
2021-08-03 15:28:51,700:INFO:Uploading model into container now
2021-08-03 15:28:51,700:INFO:create_model_container: 3
2021-08-03 15:28:51,700:INFO:master_model_container: 3
2021-08-03 15:28:51,700:INFO:display_container: 4
2021-08-03 15:28:51,700:INFO:LogisticRegression(C=1.0, class_weight=None, dual=False, fit_intercept=True,
                   intercept_scaling=1, l1_ratio=None, max_iter=1000,
                   multi_class='auto', n_jobs=None, penalty='l2',
                   random_state=3723, solver='lbfgs', tol=0.0001, verbose=0,
                   warm_start=False)
2021-08-03 15:28:51,700:INFO:create_model() succesfully completed......................................
2021-08-03 15:28:51,766:INFO:SubProcess create_model() end ==================================
2021-08-03 15:28:51,766:INFO:Creating metrics dataframe
2021-08-03 15:28:51,786:INFO:Initializing K Neighbors Classifier
2021-08-03 15:28:51,801:INFO:Total runtime is 0.0031434178352355954 minutes
2021-08-03 15:28:51,810:INFO:SubProcess create_model() called ==================================
2021-08-03 15:28:51,819:INFO:Initializing create_model()
2021-08-03 15:28:51,820:INFO:create_model(estimator=knn, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:28:51,820:INFO:Checking exceptions
2021-08-03 15:28:51,820:INFO:Importing libraries
2021-08-03 15:28:51,820:INFO:Copying training dataset
2021-08-03 15:28:51,821:INFO:Defining folds
2021-08-03 15:28:51,821:INFO:Declaring metric variables
2021-08-03 15:28:51,829:INFO:Importing untrained model
2021-08-03 15:28:51,837:INFO:K Neighbors Classifier Imported succesfully
2021-08-03 15:28:51,852:INFO:Starting cross validation
2021-08-03 15:28:51,852:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:28:52,067:WARNING:create_model() for knn raised an exception or returned all 0.0, trying without fit_kwargs:
2021-08-03 15:28:52,071:WARNING:joblib.externals.loky.process_executor._RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/externals/loky/process_executor.py", line 431, in _process_worker
    r = call_item()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/externals/loky/process_executor.py", line 285, in __call__
    return self.fn(*self.args, **self.kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/_parallel_backends.py", line 595, in __call__
    return self.func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 262, in __call__
    return [func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 262, in <listcomp>
    return [func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 560, in _fit_and_score
    test_scores = _score(estimator, X_test, y_test, scorer)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 607, in _score
    scores = scorer(estimator, X_test, y_test)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_scorer.py", line 87, in __call__
    score = scorer._score(cached_call, estimator,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_scorer.py", line 212, in _score
    return self._sign * self._score_func(y_true, y_pred,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/metrics.py", line 14, in __call__
    return self.score_func(y_true, y_pred, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1735, in recall_score
    _, r, _, _ = precision_recall_fscore_support(y_true, y_pred,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1433, in precision_recall_fscore_support
    labels = _check_set_wise_labels(y_true, y_pred, average, labels,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1263, in _check_set_wise_labels
    raise ValueError("Target is %s but average='binary'. Please "
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/tabular.py", line 2166, in compare_models
    model, model_fit_time = create_model_supervised(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/tabular.py", line 3115, in create_model_supervised
    scores = cross_validate(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 242, in cross_validate
    scores = parallel(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 1054, in __call__
    self.retrieve()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 933, in retrieve
    self._output.extend(job.get(timeout=self.timeout))
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/_parallel_backends.py", line 542, in wrap_future_result
    return future.result(timeout=timeout)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/concurrent/futures/_base.py", line 439, in result
    return self.__get_result()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/concurrent/futures/_base.py", line 388, in __get_result
    raise self._exception
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].

2021-08-03 15:28:52,071:INFO:Initializing create_model()
2021-08-03 15:28:52,071:INFO:create_model(estimator=knn, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs=None, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:28:52,071:INFO:Checking exceptions
2021-08-03 15:28:52,071:INFO:Importing libraries
2021-08-03 15:28:52,072:INFO:Copying training dataset
2021-08-03 15:28:52,072:INFO:Defining folds
2021-08-03 15:28:52,072:INFO:Declaring metric variables
2021-08-03 15:28:52,080:INFO:Importing untrained model
2021-08-03 15:28:52,089:INFO:K Neighbors Classifier Imported succesfully
2021-08-03 15:28:52,103:INFO:Starting cross validation
2021-08-03 15:28:52,104:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:28:53,848:ERROR:create_model() for knn raised an exception:
2021-08-03 15:28:53,849:ERROR:joblib.externals.loky.process_executor._RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/externals/loky/process_executor.py", line 431, in _process_worker
    r = call_item()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/externals/loky/process_executor.py", line 285, in __call__
    return self.fn(*self.args, **self.kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/_parallel_backends.py", line 595, in __call__
    return self.func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 262, in __call__
    return [func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 262, in <listcomp>
    return [func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 560, in _fit_and_score
    test_scores = _score(estimator, X_test, y_test, scorer)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 607, in _score
    scores = scorer(estimator, X_test, y_test)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_scorer.py", line 87, in __call__
    score = scorer._score(cached_call, estimator,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_scorer.py", line 212, in _score
    return self._sign * self._score_func(y_true, y_pred,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/metrics.py", line 14, in __call__
    return self.score_func(y_true, y_pred, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1735, in recall_score
    _, r, _, _ = precision_recall_fscore_support(y_true, y_pred,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1433, in precision_recall_fscore_support
    labels = _check_set_wise_labels(y_true, y_pred, average, labels,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1263, in _check_set_wise_labels
    raise ValueError("Target is %s but average='binary'. Please "
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/tabular.py", line 2186, in compare_models
    model, model_fit_time = create_model_supervised(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/tabular.py", line 3115, in create_model_supervised
    scores = cross_validate(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 242, in cross_validate
    scores = parallel(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 1054, in __call__
    self.retrieve()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 933, in retrieve
    self._output.extend(job.get(timeout=self.timeout))
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/_parallel_backends.py", line 542, in wrap_future_result
    return future.result(timeout=timeout)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/concurrent/futures/_base.py", line 432, in result
    return self.__get_result()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/concurrent/futures/_base.py", line 388, in __get_result
    raise self._exception
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].

2021-08-03 15:28:53,849:INFO:Initializing Naive Bayes
2021-08-03 15:28:53,849:INFO:Total runtime is 0.03728760083516439 minutes
2021-08-03 15:28:53,860:INFO:SubProcess create_model() called ==================================
2021-08-03 15:28:53,860:INFO:Initializing create_model()
2021-08-03 15:28:53,860:INFO:create_model(estimator=nb, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:28:53,860:INFO:Checking exceptions
2021-08-03 15:28:53,860:INFO:Importing libraries
2021-08-03 15:28:53,860:INFO:Copying training dataset
2021-08-03 15:28:53,861:INFO:Defining folds
2021-08-03 15:28:53,861:INFO:Declaring metric variables
2021-08-03 15:28:53,868:INFO:Importing untrained model
2021-08-03 15:28:53,875:INFO:Naive Bayes Imported succesfully
2021-08-03 15:28:53,889:INFO:Starting cross validation
2021-08-03 15:28:53,889:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:28:55,566:WARNING:create_model() for nb raised an exception or returned all 0.0, trying without fit_kwargs:
2021-08-03 15:28:55,567:WARNING:joblib.externals.loky.process_executor._RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/externals/loky/process_executor.py", line 431, in _process_worker
    r = call_item()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/externals/loky/process_executor.py", line 285, in __call__
    return self.fn(*self.args, **self.kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/_parallel_backends.py", line 595, in __call__
    return self.func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 262, in __call__
    return [func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 262, in <listcomp>
    return [func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 560, in _fit_and_score
    test_scores = _score(estimator, X_test, y_test, scorer)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 607, in _score
    scores = scorer(estimator, X_test, y_test)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_scorer.py", line 87, in __call__
    score = scorer._score(cached_call, estimator,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_scorer.py", line 212, in _score
    return self._sign * self._score_func(y_true, y_pred,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/metrics.py", line 14, in __call__
    return self.score_func(y_true, y_pred, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1735, in recall_score
    _, r, _, _ = precision_recall_fscore_support(y_true, y_pred,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1433, in precision_recall_fscore_support
    labels = _check_set_wise_labels(y_true, y_pred, average, labels,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1263, in _check_set_wise_labels
    raise ValueError("Target is %s but average='binary'. Please "
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/tabular.py", line 2166, in compare_models
    model, model_fit_time = create_model_supervised(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/tabular.py", line 3115, in create_model_supervised
    scores = cross_validate(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 242, in cross_validate
    scores = parallel(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 1054, in __call__
    self.retrieve()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 933, in retrieve
    self._output.extend(job.get(timeout=self.timeout))
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/_parallel_backends.py", line 542, in wrap_future_result
    return future.result(timeout=timeout)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/concurrent/futures/_base.py", line 432, in result
    return self.__get_result()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/concurrent/futures/_base.py", line 388, in __get_result
    raise self._exception
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].

2021-08-03 15:28:55,568:INFO:Initializing create_model()
2021-08-03 15:28:55,568:INFO:create_model(estimator=nb, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs=None, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:28:55,568:INFO:Checking exceptions
2021-08-03 15:28:55,568:INFO:Importing libraries
2021-08-03 15:28:55,568:INFO:Copying training dataset
2021-08-03 15:28:55,570:INFO:Defining folds
2021-08-03 15:28:55,570:INFO:Declaring metric variables
2021-08-03 15:28:55,579:INFO:Importing untrained model
2021-08-03 15:28:55,586:INFO:Naive Bayes Imported succesfully
2021-08-03 15:28:55,599:INFO:Starting cross validation
2021-08-03 15:28:55,600:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:28:57,214:ERROR:create_model() for nb raised an exception:
2021-08-03 15:28:57,215:ERROR:joblib.externals.loky.process_executor._RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/externals/loky/process_executor.py", line 431, in _process_worker
    r = call_item()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/externals/loky/process_executor.py", line 285, in __call__
    return self.fn(*self.args, **self.kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/_parallel_backends.py", line 595, in __call__
    return self.func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 262, in __call__
    return [func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 262, in <listcomp>
    return [func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 560, in _fit_and_score
    test_scores = _score(estimator, X_test, y_test, scorer)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 607, in _score
    scores = scorer(estimator, X_test, y_test)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_scorer.py", line 87, in __call__
    score = scorer._score(cached_call, estimator,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_scorer.py", line 212, in _score
    return self._sign * self._score_func(y_true, y_pred,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/metrics.py", line 14, in __call__
    return self.score_func(y_true, y_pred, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1735, in recall_score
    _, r, _, _ = precision_recall_fscore_support(y_true, y_pred,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1433, in precision_recall_fscore_support
    labels = _check_set_wise_labels(y_true, y_pred, average, labels,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1263, in _check_set_wise_labels
    raise ValueError("Target is %s but average='binary'. Please "
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/tabular.py", line 2186, in compare_models
    model, model_fit_time = create_model_supervised(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/tabular.py", line 3115, in create_model_supervised
    scores = cross_validate(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 242, in cross_validate
    scores = parallel(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 1054, in __call__
    self.retrieve()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 933, in retrieve
    self._output.extend(job.get(timeout=self.timeout))
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/_parallel_backends.py", line 542, in wrap_future_result
    return future.result(timeout=timeout)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/concurrent/futures/_base.py", line 432, in result
    return self.__get_result()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/concurrent/futures/_base.py", line 388, in __get_result
    raise self._exception
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].

2021-08-03 15:28:57,216:INFO:Initializing Decision Tree Classifier
2021-08-03 15:28:57,216:INFO:Total runtime is 0.09339459737141928 minutes
2021-08-03 15:28:57,227:INFO:SubProcess create_model() called ==================================
2021-08-03 15:28:57,227:INFO:Initializing create_model()
2021-08-03 15:28:57,227:INFO:create_model(estimator=dt, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:28:57,227:INFO:Checking exceptions
2021-08-03 15:28:57,228:INFO:Importing libraries
2021-08-03 15:28:57,228:INFO:Copying training dataset
2021-08-03 15:28:57,229:INFO:Defining folds
2021-08-03 15:28:57,229:INFO:Declaring metric variables
2021-08-03 15:28:57,235:INFO:Importing untrained model
2021-08-03 15:28:57,243:INFO:Decision Tree Classifier Imported succesfully
2021-08-03 15:28:57,256:INFO:Starting cross validation
2021-08-03 15:28:57,257:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:28:58,757:INFO:Calculating mean and std
2021-08-03 15:28:58,761:INFO:Creating metrics dataframe
2021-08-03 15:28:58,769:INFO:Uploading results into container
2021-08-03 15:28:58,769:INFO:Uploading model into container now
2021-08-03 15:28:58,769:INFO:create_model_container: 4
2021-08-03 15:28:58,769:INFO:master_model_container: 4
2021-08-03 15:28:58,769:INFO:display_container: 4
2021-08-03 15:28:58,770:INFO:DecisionTreeClassifier(ccp_alpha=0.0, class_weight=None, criterion='gini',
                       max_depth=None, max_features=None, max_leaf_nodes=None,
                       min_impurity_decrease=0.0, min_impurity_split=None,
                       min_samples_leaf=1, min_samples_split=2,
                       min_weight_fraction_leaf=0.0, presort='deprecated',
                       random_state=3723, splitter='best')
2021-08-03 15:28:58,770:INFO:create_model() succesfully completed......................................
2021-08-03 15:28:58,863:INFO:SubProcess create_model() end ==================================
2021-08-03 15:28:58,863:INFO:Creating metrics dataframe
2021-08-03 15:28:58,885:INFO:Initializing SVM - Linear Kernel
2021-08-03 15:28:58,885:INFO:Total runtime is 0.12122021516164144 minutes
2021-08-03 15:28:58,894:INFO:SubProcess create_model() called ==================================
2021-08-03 15:28:58,894:INFO:Initializing create_model()
2021-08-03 15:28:58,894:INFO:create_model(estimator=svm, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:28:58,894:INFO:Checking exceptions
2021-08-03 15:28:58,894:INFO:Importing libraries
2021-08-03 15:28:58,895:INFO:Copying training dataset
2021-08-03 15:28:58,895:INFO:Defining folds
2021-08-03 15:28:58,896:INFO:Declaring metric variables
2021-08-03 15:28:58,904:INFO:Importing untrained model
2021-08-03 15:28:58,913:INFO:SVM - Linear Kernel Imported succesfully
2021-08-03 15:28:58,928:INFO:Starting cross validation
2021-08-03 15:28:58,929:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:29:00,009:INFO:Calculating mean and std
2021-08-03 15:29:00,010:INFO:Creating metrics dataframe
2021-08-03 15:29:00,013:INFO:Uploading results into container
2021-08-03 15:29:00,013:INFO:Uploading model into container now
2021-08-03 15:29:00,013:INFO:create_model_container: 5
2021-08-03 15:29:00,014:INFO:master_model_container: 5
2021-08-03 15:29:00,014:INFO:display_container: 4
2021-08-03 15:29:00,014:INFO:SGDClassifier(alpha=0.0001, average=False, class_weight=None,
              early_stopping=False, epsilon=0.1, eta0=0.001, fit_intercept=True,
              l1_ratio=0.15, learning_rate='optimal', loss='hinge',
              max_iter=1000, n_iter_no_change=5, n_jobs=-1, penalty='l2',
              power_t=0.5, random_state=3723, shuffle=True, tol=0.001,
              validation_fraction=0.1, verbose=0, warm_start=False)
2021-08-03 15:29:00,014:INFO:create_model() succesfully completed......................................
2021-08-03 15:29:00,085:INFO:SubProcess create_model() end ==================================
2021-08-03 15:29:00,085:INFO:Creating metrics dataframe
2021-08-03 15:29:00,109:INFO:Initializing Ridge Classifier
2021-08-03 15:29:00,109:INFO:Total runtime is 0.14161210060119628 minutes
2021-08-03 15:29:00,117:INFO:SubProcess create_model() called ==================================
2021-08-03 15:29:00,118:INFO:Initializing create_model()
2021-08-03 15:29:00,118:INFO:create_model(estimator=ridge, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:29:00,118:INFO:Checking exceptions
2021-08-03 15:29:00,118:INFO:Importing libraries
2021-08-03 15:29:00,118:INFO:Copying training dataset
2021-08-03 15:29:00,119:INFO:Defining folds
2021-08-03 15:29:00,119:INFO:Declaring metric variables
2021-08-03 15:29:00,127:INFO:Importing untrained model
2021-08-03 15:29:00,136:INFO:Ridge Classifier Imported succesfully
2021-08-03 15:29:00,151:INFO:Starting cross validation
2021-08-03 15:29:00,152:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:29:01,358:INFO:Calculating mean and std
2021-08-03 15:29:01,359:INFO:Creating metrics dataframe
2021-08-03 15:29:01,364:INFO:Uploading results into container
2021-08-03 15:29:01,364:INFO:Uploading model into container now
2021-08-03 15:29:01,364:INFO:create_model_container: 6
2021-08-03 15:29:01,364:INFO:master_model_container: 6
2021-08-03 15:29:01,364:INFO:display_container: 4
2021-08-03 15:29:01,365:INFO:RidgeClassifier(alpha=1.0, class_weight=None, copy_X=True, fit_intercept=True,
                max_iter=None, normalize=False, random_state=3723,
                solver='auto', tol=0.001)
2021-08-03 15:29:01,365:INFO:create_model() succesfully completed......................................
2021-08-03 15:29:01,444:INFO:SubProcess create_model() end ==================================
2021-08-03 15:29:01,445:INFO:Creating metrics dataframe
2021-08-03 15:29:01,469:INFO:Initializing Random Forest Classifier
2021-08-03 15:29:01,469:INFO:Total runtime is 0.16428578297297158 minutes
2021-08-03 15:29:01,478:INFO:SubProcess create_model() called ==================================
2021-08-03 15:29:01,478:INFO:Initializing create_model()
2021-08-03 15:29:01,478:INFO:create_model(estimator=rf, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:29:01,478:INFO:Checking exceptions
2021-08-03 15:29:01,478:INFO:Importing libraries
2021-08-03 15:29:01,478:INFO:Copying training dataset
2021-08-03 15:29:01,479:INFO:Defining folds
2021-08-03 15:29:01,479:INFO:Declaring metric variables
2021-08-03 15:29:01,487:INFO:Importing untrained model
2021-08-03 15:29:01,496:INFO:Random Forest Classifier Imported succesfully
2021-08-03 15:29:01,511:INFO:Starting cross validation
2021-08-03 15:29:01,512:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:29:02,772:INFO:Calculating mean and std
2021-08-03 15:29:02,773:INFO:Creating metrics dataframe
2021-08-03 15:29:02,776:INFO:Uploading results into container
2021-08-03 15:29:02,777:INFO:Uploading model into container now
2021-08-03 15:29:02,777:INFO:create_model_container: 7
2021-08-03 15:29:02,777:INFO:master_model_container: 7
2021-08-03 15:29:02,777:INFO:display_container: 4
2021-08-03 15:29:02,777:INFO:RandomForestClassifier(bootstrap=True, ccp_alpha=0.0, class_weight=None,
                       criterion='gini', max_depth=None, max_features='auto',
                       max_leaf_nodes=None, max_samples=None,
                       min_impurity_decrease=0.0, min_impurity_split=None,
                       min_samples_leaf=1, min_samples_split=2,
                       min_weight_fraction_leaf=0.0, n_estimators=100,
                       n_jobs=-1, oob_score=False, random_state=3723, verbose=0,
                       warm_start=False)
2021-08-03 15:29:02,777:INFO:create_model() succesfully completed......................................
2021-08-03 15:29:02,846:INFO:SubProcess create_model() end ==================================
2021-08-03 15:29:02,846:INFO:Creating metrics dataframe
2021-08-03 15:29:02,870:INFO:Initializing Quadratic Discriminant Analysis
2021-08-03 15:29:02,870:INFO:Total runtime is 0.187629501024882 minutes
2021-08-03 15:29:02,878:INFO:SubProcess create_model() called ==================================
2021-08-03 15:29:02,879:INFO:Initializing create_model()
2021-08-03 15:29:02,879:INFO:create_model(estimator=qda, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:29:02,879:INFO:Checking exceptions
2021-08-03 15:29:02,879:INFO:Importing libraries
2021-08-03 15:29:02,879:INFO:Copying training dataset
2021-08-03 15:29:02,880:INFO:Defining folds
2021-08-03 15:29:02,880:INFO:Declaring metric variables
2021-08-03 15:29:02,887:INFO:Importing untrained model
2021-08-03 15:29:02,896:INFO:Quadratic Discriminant Analysis Imported succesfully
2021-08-03 15:29:02,911:INFO:Starting cross validation
2021-08-03 15:29:02,911:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:29:02,967:INFO:Calculating mean and std
2021-08-03 15:29:02,967:INFO:Creating metrics dataframe
2021-08-03 15:29:02,972:INFO:Uploading results into container
2021-08-03 15:29:02,972:INFO:Uploading model into container now
2021-08-03 15:29:02,972:INFO:create_model_container: 8
2021-08-03 15:29:02,972:INFO:master_model_container: 8
2021-08-03 15:29:02,972:INFO:display_container: 4
2021-08-03 15:29:02,972:INFO:QuadraticDiscriminantAnalysis(priors=None, reg_param=0.0,
                              store_covariance=False, tol=0.0001)
2021-08-03 15:29:02,972:INFO:create_model() succesfully completed......................................
2021-08-03 15:29:03,036:WARNING:create_model() for QuadraticDiscriminantAnalysis(priors=None, reg_param=0.0,
                              store_covariance=False, tol=0.0001) raised an exception or returned all 0.0, trying without fit_kwargs:
2021-08-03 15:29:03,036:WARNING:Traceback (most recent call last):
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/tabular.py", line 2179, in compare_models
    assert np.sum(model_results.iloc[0]) != 0.0
AssertionError

2021-08-03 15:29:03,037:INFO:Initializing create_model()
2021-08-03 15:29:03,037:INFO:create_model(estimator=QuadraticDiscriminantAnalysis(priors=None, reg_param=0.0,
                              store_covariance=False, tol=0.0001), fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs=None, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:29:03,037:INFO:Checking exceptions
2021-08-03 15:29:03,037:INFO:Importing libraries
2021-08-03 15:29:03,037:INFO:Copying training dataset
2021-08-03 15:29:03,038:INFO:Defining folds
2021-08-03 15:29:03,038:INFO:Declaring metric variables
2021-08-03 15:29:03,047:INFO:Importing untrained model
2021-08-03 15:29:03,047:INFO:Declaring custom model
2021-08-03 15:29:03,056:INFO:Quadratic Discriminant Analysis Imported succesfully
2021-08-03 15:29:03,073:INFO:Starting cross validation
2021-08-03 15:29:03,074:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:29:04,262:INFO:Calculating mean and std
2021-08-03 15:29:04,263:INFO:Creating metrics dataframe
2021-08-03 15:29:04,266:INFO:Uploading results into container
2021-08-03 15:29:04,266:INFO:Uploading model into container now
2021-08-03 15:29:04,266:INFO:create_model_container: 9
2021-08-03 15:29:04,266:INFO:master_model_container: 9
2021-08-03 15:29:04,267:INFO:display_container: 4
2021-08-03 15:29:04,267:INFO:QuadraticDiscriminantAnalysis(priors=None, reg_param=0.0,
                              store_covariance=False, tol=0.0001)
2021-08-03 15:29:04,267:INFO:create_model() succesfully completed......................................
2021-08-03 15:29:04,345:INFO:SubProcess create_model() end ==================================
2021-08-03 15:29:04,345:INFO:Creating metrics dataframe
2021-08-03 15:29:04,369:INFO:Initializing Ada Boost Classifier
2021-08-03 15:29:04,369:INFO:Total runtime is 0.21261481841405233 minutes
2021-08-03 15:29:04,377:INFO:SubProcess create_model() called ==================================
2021-08-03 15:29:04,378:INFO:Initializing create_model()
2021-08-03 15:29:04,378:INFO:create_model(estimator=ada, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:29:04,378:INFO:Checking exceptions
2021-08-03 15:29:04,378:INFO:Importing libraries
2021-08-03 15:29:04,378:INFO:Copying training dataset
2021-08-03 15:29:04,379:INFO:Defining folds
2021-08-03 15:29:04,379:INFO:Declaring metric variables
2021-08-03 15:29:04,386:INFO:Importing untrained model
2021-08-03 15:29:04,394:INFO:Ada Boost Classifier Imported succesfully
2021-08-03 15:29:04,408:INFO:Starting cross validation
2021-08-03 15:29:04,409:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:29:05,632:INFO:Calculating mean and std
2021-08-03 15:29:05,632:INFO:Creating metrics dataframe
2021-08-03 15:29:05,636:INFO:Uploading results into container
2021-08-03 15:29:05,636:INFO:Uploading model into container now
2021-08-03 15:29:05,636:INFO:create_model_container: 10
2021-08-03 15:29:05,636:INFO:master_model_container: 10
2021-08-03 15:29:05,636:INFO:display_container: 4
2021-08-03 15:29:05,636:INFO:AdaBoostClassifier(algorithm='SAMME.R', base_estimator=None, learning_rate=1.0,
                   n_estimators=50, random_state=3723)
2021-08-03 15:29:05,636:INFO:create_model() succesfully completed......................................
2021-08-03 15:29:05,704:INFO:SubProcess create_model() end ==================================
2021-08-03 15:29:05,704:INFO:Creating metrics dataframe
2021-08-03 15:29:05,729:INFO:Initializing Gradient Boosting Classifier
2021-08-03 15:29:05,729:INFO:Total runtime is 0.23528533379236857 minutes
2021-08-03 15:29:05,738:INFO:SubProcess create_model() called ==================================
2021-08-03 15:29:05,738:INFO:Initializing create_model()
2021-08-03 15:29:05,738:INFO:create_model(estimator=gbc, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:29:05,738:INFO:Checking exceptions
2021-08-03 15:29:05,738:INFO:Importing libraries
2021-08-03 15:29:05,738:INFO:Copying training dataset
2021-08-03 15:29:05,739:INFO:Defining folds
2021-08-03 15:29:05,740:INFO:Declaring metric variables
2021-08-03 15:29:05,748:INFO:Importing untrained model
2021-08-03 15:29:05,756:INFO:Gradient Boosting Classifier Imported succesfully
2021-08-03 15:29:05,772:INFO:Starting cross validation
2021-08-03 15:29:05,772:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:29:06,992:INFO:Calculating mean and std
2021-08-03 15:29:06,993:INFO:Creating metrics dataframe
2021-08-03 15:29:06,996:INFO:Uploading results into container
2021-08-03 15:29:06,996:INFO:Uploading model into container now
2021-08-03 15:29:06,996:INFO:create_model_container: 11
2021-08-03 15:29:06,997:INFO:master_model_container: 11
2021-08-03 15:29:06,997:INFO:display_container: 4
2021-08-03 15:29:06,997:INFO:GradientBoostingClassifier(ccp_alpha=0.0, criterion='friedman_mse', init=None,
                           learning_rate=0.1, loss='deviance', max_depth=3,
                           max_features=None, max_leaf_nodes=None,
                           min_impurity_decrease=0.0, min_impurity_split=None,
                           min_samples_leaf=1, min_samples_split=2,
                           min_weight_fraction_leaf=0.0, n_estimators=100,
                           n_iter_no_change=None, presort='deprecated',
                           random_state=3723, subsample=1.0, tol=0.0001,
                           validation_fraction=0.1, verbose=0,
                           warm_start=False)
2021-08-03 15:29:06,997:INFO:create_model() succesfully completed......................................
2021-08-03 15:29:07,064:INFO:SubProcess create_model() end ==================================
2021-08-03 15:29:07,064:INFO:Creating metrics dataframe
2021-08-03 15:29:07,089:INFO:Initializing Linear Discriminant Analysis
2021-08-03 15:29:07,089:INFO:Total runtime is 0.2579527179400126 minutes
2021-08-03 15:29:07,097:INFO:SubProcess create_model() called ==================================
2021-08-03 15:29:07,098:INFO:Initializing create_model()
2021-08-03 15:29:07,098:INFO:create_model(estimator=lda, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:29:07,098:INFO:Checking exceptions
2021-08-03 15:29:07,098:INFO:Importing libraries
2021-08-03 15:29:07,098:INFO:Copying training dataset
2021-08-03 15:29:07,099:INFO:Defining folds
2021-08-03 15:29:07,099:INFO:Declaring metric variables
2021-08-03 15:29:07,106:INFO:Importing untrained model
2021-08-03 15:29:07,114:INFO:Linear Discriminant Analysis Imported succesfully
2021-08-03 15:29:07,129:INFO:Starting cross validation
2021-08-03 15:29:07,129:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:29:08,437:WARNING:create_model() for lda raised an exception or returned all 0.0, trying without fit_kwargs:
2021-08-03 15:29:08,437:WARNING:joblib.externals.loky.process_executor._RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/externals/loky/process_executor.py", line 431, in _process_worker
    r = call_item()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/externals/loky/process_executor.py", line 285, in __call__
    return self.fn(*self.args, **self.kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/_parallel_backends.py", line 595, in __call__
    return self.func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 262, in __call__
    return [func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 262, in <listcomp>
    return [func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 560, in _fit_and_score
    test_scores = _score(estimator, X_test, y_test, scorer)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 607, in _score
    scores = scorer(estimator, X_test, y_test)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_scorer.py", line 87, in __call__
    score = scorer._score(cached_call, estimator,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_scorer.py", line 212, in _score
    return self._sign * self._score_func(y_true, y_pred,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/metrics.py", line 14, in __call__
    return self.score_func(y_true, y_pred, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1735, in recall_score
    _, r, _, _ = precision_recall_fscore_support(y_true, y_pred,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1433, in precision_recall_fscore_support
    labels = _check_set_wise_labels(y_true, y_pred, average, labels,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1263, in _check_set_wise_labels
    raise ValueError("Target is %s but average='binary'. Please "
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/tabular.py", line 2166, in compare_models
    model, model_fit_time = create_model_supervised(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/tabular.py", line 3115, in create_model_supervised
    scores = cross_validate(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 242, in cross_validate
    scores = parallel(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 1054, in __call__
    self.retrieve()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 933, in retrieve
    self._output.extend(job.get(timeout=self.timeout))
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/_parallel_backends.py", line 542, in wrap_future_result
    return future.result(timeout=timeout)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/concurrent/futures/_base.py", line 432, in result
    return self.__get_result()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/concurrent/futures/_base.py", line 388, in __get_result
    raise self._exception
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].

2021-08-03 15:29:08,437:INFO:Initializing create_model()
2021-08-03 15:29:08,438:INFO:create_model(estimator=lda, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs=None, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:29:08,438:INFO:Checking exceptions
2021-08-03 15:29:08,438:INFO:Importing libraries
2021-08-03 15:29:08,438:INFO:Copying training dataset
2021-08-03 15:29:08,438:INFO:Defining folds
2021-08-03 15:29:08,438:INFO:Declaring metric variables
2021-08-03 15:29:08,445:INFO:Importing untrained model
2021-08-03 15:29:08,452:INFO:Linear Discriminant Analysis Imported succesfully
2021-08-03 15:29:08,465:INFO:Starting cross validation
2021-08-03 15:29:08,466:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:29:10,130:ERROR:create_model() for lda raised an exception:
2021-08-03 15:29:10,132:ERROR:joblib.externals.loky.process_executor._RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/externals/loky/process_executor.py", line 431, in _process_worker
    r = call_item()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/externals/loky/process_executor.py", line 285, in __call__
    return self.fn(*self.args, **self.kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/_parallel_backends.py", line 595, in __call__
    return self.func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 262, in __call__
    return [func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 262, in <listcomp>
    return [func(*args, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 560, in _fit_and_score
    test_scores = _score(estimator, X_test, y_test, scorer)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 607, in _score
    scores = scorer(estimator, X_test, y_test)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_scorer.py", line 87, in __call__
    score = scorer._score(cached_call, estimator,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_scorer.py", line 212, in _score
    return self._sign * self._score_func(y_true, y_pred,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/metrics.py", line 14, in __call__
    return self.score_func(y_true, y_pred, **kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1735, in recall_score
    _, r, _, _ = precision_recall_fscore_support(y_true, y_pred,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1433, in precision_recall_fscore_support
    labels = _check_set_wise_labels(y_true, y_pred, average, labels,
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 1263, in _check_set_wise_labels
    raise ValueError("Target is %s but average='binary'. Please "
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/tabular.py", line 2186, in compare_models
    model, model_fit_time = create_model_supervised(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/pycaret/internal/tabular.py", line 3115, in create_model_supervised
    scores = cross_validate(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 242, in cross_validate
    scores = parallel(
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 1054, in __call__
    self.retrieve()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/parallel.py", line 933, in retrieve
    self._output.extend(job.get(timeout=self.timeout))
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/site-packages/joblib/_parallel_backends.py", line 542, in wrap_future_result
    return future.result(timeout=timeout)
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/concurrent/futures/_base.py", line 432, in result
    return self.__get_result()
  File "/Users/ceyhun/opt/anaconda3/lib/python3.8/concurrent/futures/_base.py", line 388, in __get_result
    raise self._exception
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].

2021-08-03 15:29:10,132:INFO:Initializing Extra Trees Classifier
2021-08-03 15:29:10,132:INFO:Total runtime is 0.3086669325828552 minutes
2021-08-03 15:29:10,144:INFO:SubProcess create_model() called ==================================
2021-08-03 15:29:10,144:INFO:Initializing create_model()
2021-08-03 15:29:10,144:INFO:create_model(estimator=et, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:29:10,144:INFO:Checking exceptions
2021-08-03 15:29:10,144:INFO:Importing libraries
2021-08-03 15:29:10,144:INFO:Copying training dataset
2021-08-03 15:29:10,145:INFO:Defining folds
2021-08-03 15:29:10,145:INFO:Declaring metric variables
2021-08-03 15:29:10,152:INFO:Importing untrained model
2021-08-03 15:29:10,159:INFO:Extra Trees Classifier Imported succesfully
2021-08-03 15:29:10,172:INFO:Starting cross validation
2021-08-03 15:29:10,173:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:29:11,923:INFO:Calculating mean and std
2021-08-03 15:29:11,927:INFO:Creating metrics dataframe
2021-08-03 15:29:11,934:INFO:Uploading results into container
2021-08-03 15:29:11,934:INFO:Uploading model into container now
2021-08-03 15:29:11,934:INFO:create_model_container: 12
2021-08-03 15:29:11,934:INFO:master_model_container: 12
2021-08-03 15:29:11,934:INFO:display_container: 4
2021-08-03 15:29:11,935:INFO:ExtraTreesClassifier(bootstrap=False, ccp_alpha=0.0, class_weight=None,
                     criterion='gini', max_depth=None, max_features='auto',
                     max_leaf_nodes=None, max_samples=None,
                     min_impurity_decrease=0.0, min_impurity_split=None,
                     min_samples_leaf=1, min_samples_split=2,
                     min_weight_fraction_leaf=0.0, n_estimators=100, n_jobs=-1,
                     oob_score=False, random_state=3723, verbose=0,
                     warm_start=False)
2021-08-03 15:29:11,935:INFO:create_model() succesfully completed......................................
2021-08-03 15:29:12,033:INFO:SubProcess create_model() end ==================================
2021-08-03 15:29:12,033:INFO:Creating metrics dataframe
2021-08-03 15:29:12,060:INFO:Initializing Extreme Gradient Boosting
2021-08-03 15:29:12,060:INFO:Total runtime is 0.34079816738764446 minutes
2021-08-03 15:29:12,069:INFO:SubProcess create_model() called ==================================
2021-08-03 15:29:12,069:INFO:Initializing create_model()
2021-08-03 15:29:12,069:INFO:create_model(estimator=xgboost, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:29:12,069:INFO:Checking exceptions
2021-08-03 15:29:12,069:INFO:Importing libraries
2021-08-03 15:29:12,070:INFO:Copying training dataset
2021-08-03 15:29:12,070:INFO:Defining folds
2021-08-03 15:29:12,071:INFO:Declaring metric variables
2021-08-03 15:29:12,078:INFO:Importing untrained model
2021-08-03 15:29:12,088:INFO:Extreme Gradient Boosting Imported succesfully
2021-08-03 15:29:12,103:INFO:Starting cross validation
2021-08-03 15:29:12,105:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:29:13,875:INFO:Calculating mean and std
2021-08-03 15:29:13,876:INFO:Creating metrics dataframe
2021-08-03 15:29:13,882:INFO:Uploading results into container
2021-08-03 15:29:13,882:INFO:Uploading model into container now
2021-08-03 15:29:13,882:INFO:create_model_container: 13
2021-08-03 15:29:13,882:INFO:master_model_container: 13
2021-08-03 15:29:13,883:INFO:display_container: 4
2021-08-03 15:29:13,883:INFO:XGBClassifier(base_score=None, booster='gbtree', colsample_bylevel=None,
              colsample_bynode=None, colsample_bytree=None, gamma=None,
              gpu_id=None, importance_type='gain', interaction_constraints=None,
              learning_rate=None, max_delta_step=None, max_depth=None,
              min_child_weight=None, missing=nan, monotone_constraints=None,
              n_estimators=100, n_jobs=-1, num_parallel_tree=None,
              objective='binary:logistic', random_state=3723, reg_alpha=None,
              reg_lambda=None, scale_pos_weight=None, subsample=None,
              tree_method='auto', use_label_encoder=True,
              validate_parameters=None, verbosity=0)
2021-08-03 15:29:13,884:INFO:create_model() succesfully completed......................................
2021-08-03 15:29:13,956:INFO:SubProcess create_model() end ==================================
2021-08-03 15:29:13,957:INFO:Creating metrics dataframe
2021-08-03 15:29:13,981:INFO:Initializing Light Gradient Boosting Machine
2021-08-03 15:29:13,981:INFO:Total runtime is 0.3728225827217102 minutes
2021-08-03 15:29:13,991:INFO:SubProcess create_model() called ==================================
2021-08-03 15:29:13,991:INFO:Initializing create_model()
2021-08-03 15:29:13,991:INFO:create_model(estimator=lightgbm, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:29:13,992:INFO:Checking exceptions
2021-08-03 15:29:13,992:INFO:Importing libraries
2021-08-03 15:29:13,992:INFO:Copying training dataset
2021-08-03 15:29:13,993:INFO:Defining folds
2021-08-03 15:29:13,993:INFO:Declaring metric variables
2021-08-03 15:29:14,002:INFO:Importing untrained model
2021-08-03 15:29:14,012:INFO:Light Gradient Boosting Machine Imported succesfully
2021-08-03 15:29:14,031:INFO:Starting cross validation
2021-08-03 15:29:14,032:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:29:15,760:INFO:Calculating mean and std
2021-08-03 15:29:15,761:INFO:Creating metrics dataframe
2021-08-03 15:29:15,765:INFO:Uploading results into container
2021-08-03 15:29:15,765:INFO:Uploading model into container now
2021-08-03 15:29:15,765:INFO:create_model_container: 14
2021-08-03 15:29:15,766:INFO:master_model_container: 14
2021-08-03 15:29:15,766:INFO:display_container: 4
2021-08-03 15:29:15,766:INFO:LGBMClassifier(boosting_type='gbdt', class_weight=None, colsample_bytree=1.0,
               importance_type='split', learning_rate=0.1, max_depth=-1,
               min_child_samples=20, min_child_weight=0.001, min_split_gain=0.0,
               n_estimators=100, n_jobs=-1, num_leaves=31, objective=None,
               random_state=3723, reg_alpha=0.0, reg_lambda=0.0, silent=True,
               subsample=1.0, subsample_for_bin=200000, subsample_freq=0)
2021-08-03 15:29:15,766:INFO:create_model() succesfully completed......................................
2021-08-03 15:29:15,844:INFO:SubProcess create_model() end ==================================
2021-08-03 15:29:15,844:INFO:Creating metrics dataframe
2021-08-03 15:29:15,872:INFO:Initializing CatBoost Classifier
2021-08-03 15:29:15,872:INFO:Total runtime is 0.4043348828951518 minutes
2021-08-03 15:29:15,881:INFO:SubProcess create_model() called ==================================
2021-08-03 15:29:15,881:INFO:Initializing create_model()
2021-08-03 15:29:15,881:INFO:create_model(estimator=catboost, fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x7fb4737a8b50>, kwargs={})
2021-08-03 15:29:15,881:INFO:Checking exceptions
2021-08-03 15:29:15,881:INFO:Importing libraries
2021-08-03 15:29:15,881:INFO:Copying training dataset
2021-08-03 15:29:15,882:INFO:Defining folds
2021-08-03 15:29:15,882:INFO:Declaring metric variables
2021-08-03 15:29:15,890:INFO:Importing untrained model
2021-08-03 15:29:15,899:INFO:CatBoost Classifier Imported succesfully
2021-08-03 15:29:15,915:INFO:Starting cross validation
2021-08-03 15:29:15,916:INFO:Cross validating with StratifiedKFold(n_splits=3, random_state=None, shuffle=False), n_jobs=-1
2021-08-03 15:29:17,910:INFO:Calculating mean and std
2021-08-03 15:29:17,910:INFO:Creating metrics dataframe
2021-08-03 15:29:17,914:INFO:Uploading results into container
2021-08-03 15:29:17,914:INFO:Uploading model into container now
2021-08-03 15:29:17,914:INFO:create_model_container: 15
2021-08-03 15:29:17,914:INFO:master_model_container: 15
2021-08-03 15:29:17,915:INFO:display_container: 4
2021-08-03 15:29:17,915:INFO:<catboost.core.CatBoostClassifier object at 0x7fb47a526910>
2021-08-03 15:29:17,915:INFO:create_model() succesfully completed......................................
2021-08-03 15:29:17,990:INFO:SubProcess create_model() end ==================================
2021-08-03 15:29:17,990:INFO:Creating metrics dataframe
2021-08-03 15:29:18,037:INFO:Initializing create_model()
2021-08-03 15:29:18,037:INFO:create_model(estimator=LGBMClassifier(boosting_type='gbdt', class_weight=None, colsample_bytree=1.0,
               importance_type='split', learning_rate=0.1, max_depth=-1,
               min_child_samples=20, min_child_weight=0.001, min_split_gain=0.0,
               n_estimators=100, n_jobs=-1, num_leaves=31, objective=None,
               random_state=3723, reg_alpha=0.0, reg_lambda=0.0, silent=True,
               subsample=1.0, subsample_for_bin=200000, subsample_freq=0), fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=False, predict=False, fit_kwargs={}, groups=None, refit=True, verbose=False, system=False, metrics=None, display=None, kwargs={})
2021-08-03 15:29:18,037:INFO:Checking exceptions
2021-08-03 15:29:18,037:INFO:Importing libraries
2021-08-03 15:29:18,037:INFO:Copying training dataset
2021-08-03 15:29:18,038:INFO:Defining folds
2021-08-03 15:29:18,038:INFO:Declaring metric variables
2021-08-03 15:29:18,038:INFO:Importing untrained model
2021-08-03 15:29:18,038:INFO:Declaring custom model
2021-08-03 15:29:18,038:INFO:Light Gradient Boosting Machine Imported succesfully
2021-08-03 15:29:18,039:INFO:Cross validation set to False
2021-08-03 15:29:18,039:INFO:Fitting Model
2021-08-03 15:29:18,054:INFO:LGBMClassifier(boosting_type='gbdt', class_weight=None, colsample_bytree=1.0,
               importance_type='split', learning_rate=0.1, max_depth=-1,
               min_child_samples=20, min_child_weight=0.001, min_split_gain=0.0,
               n_estimators=100, n_jobs=-1, num_leaves=31, objective=None,
               random_state=3723, reg_alpha=0.0, reg_lambda=0.0, silent=True,
               subsample=1.0, subsample_for_bin=200000, subsample_freq=0)
2021-08-03 15:29:18,054:INFO:create_models() succesfully completed......................................
2021-08-03 15:29:18,131:INFO:Initializing create_model()
2021-08-03 15:29:18,131:INFO:create_model(estimator=SGDClassifier(alpha=0.0001, average=False, class_weight=None,
              early_stopping=False, epsilon=0.1, eta0=0.001, fit_intercept=True,
              l1_ratio=0.15, learning_rate='optimal', loss='hinge',
              max_iter=1000, n_iter_no_change=5, n_jobs=-1, penalty='l2',
              power_t=0.5, random_state=3723, shuffle=True, tol=0.001,
              validation_fraction=0.1, verbose=0, warm_start=False), fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=False, predict=False, fit_kwargs={}, groups=None, refit=True, verbose=False, system=False, metrics=None, display=None, kwargs={})
2021-08-03 15:29:18,131:INFO:Checking exceptions
2021-08-03 15:29:18,131:INFO:Importing libraries
2021-08-03 15:29:18,131:INFO:Copying training dataset
2021-08-03 15:29:18,132:INFO:Defining folds
2021-08-03 15:29:18,132:INFO:Declaring metric variables
2021-08-03 15:29:18,132:INFO:Importing untrained model
2021-08-03 15:29:18,132:INFO:Declaring custom model
2021-08-03 15:29:18,133:INFO:SVM - Linear Kernel Imported succesfully
2021-08-03 15:29:18,133:INFO:Cross validation set to False
2021-08-03 15:29:18,134:INFO:Fitting Model
2021-08-03 15:29:18,145:INFO:SGDClassifier(alpha=0.0001, average=False, class_weight=None,
              early_stopping=False, epsilon=0.1, eta0=0.001, fit_intercept=True,
              l1_ratio=0.15, learning_rate='optimal', loss='hinge',
              max_iter=1000, n_iter_no_change=5, n_jobs=-1, penalty='l2',
              power_t=0.5, random_state=3723, shuffle=True, tol=0.001,
              validation_fraction=0.1, verbose=0, warm_start=False)
2021-08-03 15:29:18,145:INFO:create_models() succesfully completed......................................
2021-08-03 15:29:18,222:INFO:Initializing create_model()
2021-08-03 15:29:18,222:INFO:create_model(estimator=XGBClassifier(base_score=None, booster='gbtree', colsample_bylevel=None,
              colsample_bynode=None, colsample_bytree=None, gamma=None,
              gpu_id=None, importance_type='gain', interaction_constraints=None,
              learning_rate=None, max_delta_step=None, max_depth=None,
              min_child_weight=None, missing=nan, monotone_constraints=None,
              n_estimators=100, n_jobs=-1, num_parallel_tree=None,
              objective='binary:logistic', random_state=3723, reg_alpha=None,
              reg_lambda=None, scale_pos_weight=None, subsample=None,
              tree_method='auto', use_label_encoder=True,
              validate_parameters=None, verbosity=0), fold=StratifiedKFold(n_splits=3, random_state=None, shuffle=False), round=4, cross_validation=False, predict=False, fit_kwargs={}, groups=None, refit=True, verbose=False, system=False, metrics=None, display=None, kwargs={})
2021-08-03 15:29:18,222:INFO:Checking exceptions
2021-08-03 15:29:18,222:INFO:Importing libraries
2021-08-03 15:29:18,223:INFO:Copying training dataset
2021-08-03 15:29:18,223:INFO:Defining folds
2021-08-03 15:29:18,223:INFO:Declaring metric variables
2021-08-03 15:29:18,223:INFO:Importing untrained model
2021-08-03 15:29:18,223:INFO:Declaring custom model
2021-08-03 15:29:18,225:INFO:Extreme Gradient Boosting Imported succesfully
2021-08-03 15:29:18,226:INFO:Cross validation set to False
2021-08-03 15:29:18,226:INFO:Fitting Model
2021-08-03 15:29:18,337:INFO:XGBClassifier(base_score=0.5, booster='gbtree', colsample_bylevel=1,
              colsample_bynode=1, colsample_bytree=1, gamma=0, gpu_id=-1,
              importance_type='gain', interaction_constraints='',
              learning_rate=0.300000012, max_delta_step=0, max_depth=6,
              min_child_weight=1, missing=nan, monotone_constraints='()',
              n_estimators=100, n_jobs=-1, num_parallel_tree=1,
              objective='multi:softprob', random_state=3723, reg_alpha=0,
              reg_lambda=1, scale_pos_weight=None, subsample=1,
              tree_method='auto', use_label_encoder=True, validate_parameters=1,
              verbosity=0)
2021-08-03 15:29:18,338:INFO:create_models() succesfully completed......................................
2021-08-03 15:29:18,448:INFO:create_model_container: 15
2021-08-03 15:29:18,448:INFO:master_model_container: 15
2021-08-03 15:29:18,448:INFO:display_container: 4
2021-08-03 15:29:18,451:INFO:[LGBMClassifier(boosting_type='gbdt', class_weight=None, colsample_bytree=1.0,
               importance_type='split', learning_rate=0.1, max_depth=-1,
               min_child_samples=20, min_child_weight=0.001, min_split_gain=0.0,
               n_estimators=100, n_jobs=-1, num_leaves=31, objective=None,
               random_state=3723, reg_alpha=0.0, reg_lambda=0.0, silent=True,
               subsample=1.0, subsample_for_bin=200000, subsample_freq=0), SGDClassifier(alpha=0.0001, average=False, class_weight=None,
              early_stopping=False, epsilon=0.1, eta0=0.001, fit_intercept=True,
              l1_ratio=0.15, learning_rate='optimal', loss='hinge',
              max_iter=1000, n_iter_no_change=5, n_jobs=-1, penalty='l2',
              power_t=0.5, random_state=3723, shuffle=True, tol=0.001,
              validation_fraction=0.1, verbose=0, warm_start=False), XGBClassifier(base_score=0.5, booster='gbtree', colsample_bylevel=1,
              colsample_bynode=1, colsample_bytree=1, gamma=0, gpu_id=-1,
              importance_type='gain', interaction_constraints='',
              learning_rate=0.300000012, max_delta_step=0, max_depth=6,
              min_child_weight=1, missing=nan, monotone_constraints='()',
              n_estimators=100, n_jobs=-1, num_parallel_tree=1,
              objective='multi:softprob', random_state=3723, reg_alpha=0,
              reg_lambda=1, scale_pos_weight=None, subsample=1,
              tree_method='auto', use_label_encoder=True, validate_parameters=1,
              verbosity=0)]
2021-08-03 15:29:18,451:INFO:compare_models() succesfully completed......................................
